helm/cluster-vsphere/templates/cilium-helmrelease.yaml

```diff
@@ -32,6 +32,11 @@ spec:
   # Default values
   # https://github.com/giantswarm/cilium-app/blob/main/helm/cilium/values.yaml
   values:
+    {{- if semverCompare ">=1.25" .Values.cluster.kubernetesVersion }}
```

Comment:
This could be not right for the migration path. It is not fully clear how we will do the migration. I saw it was discussed to enable PSSs in 1.24 and then do upgrade by disabling PSPs. Therefore, we may need to enable PSSs for k8s 1.24.

Reply:
related https://github.com/giantswarm/roadmap/issues/2925

Reply:
Do you think it is preferable to disable it via a flag?

Reply:
Also, regardless of how we do it, it won't work on 1.25 unless this is set